avaframe/com1DFA/particleInitialisation.py

```diff
@@ -0,0 +1,246 @@
+"""
+    functions for initialising particle distribution
+"""
+
+import logging
+import time
+import pathlib
+import numpy as np
+import copy
+from shapely.geometry import Polygon
+import matplotlib.pyplot as plt
+
+# Local imports
+import avaframe.com1DFA.DFAfunctionsCython as DFAfunC
+import avaframe.out3Plot.outDebugPlots as debPlot
+import avaframe.com1DFA.com1DFA as com1DFA
+import avaframe.com1DFA.particleTools as particleTools
+
+# create local logger
+log = logging.getLogger(__name__)
+
+def getIniPosition(cfg, particles, dem, fields, inputSimLines, relThField):
+    """ Redistribute particles so that SPH force reduces with fixed particles as boundaries
+
+        Parameters
+        ------------
+        cfg: configparser object
+            configuration settings
+        particles: dict
+            dictionary with particle properties
+        dem: dict
+            dictionary with dem header and data
+        fields: dict
+            dictionary with fields of result types
+        inputSimLines: dict
+            info on input files
+        relThField: numpy array
+            release thickness field
+
+        Returns
+        --------
+        particles: dict
+            updated particles dict with new positions
+        fields: fields
+            updated fields dictionary
+        """
+
+    iterate = True
+    particles['iterate'] = True
+    particlesList = [particles.copy()]
+    countIterations = 0
+    # load relRaster from buffered release line
+    relRaster = inputSimLines['releaseLineBuffer']['rasterData']
+
+    # redistribute particles to reduce SPH force
+    while iterate and countIterations < cfg['GENERAL'].getint('maxIterations'):
```

Comment:
I suggest dropping the variable `iterate` and just use `particles['iterate']`

Reply:
if you do it like this please also do the same in DFAiterate

Reply:
done